evennia/server/tests.py

```diff
@@ -46,3 +43,37 @@ def build_suite(self, test_labels, extra_tests=None, **kwargs):
         import evennia
         evennia._init()
         return super(EvenniaTestSuiteRunner, self).build_suite(test_labels, extra_tests=extra_tests, **kwargs)
+
+
+class MockSettings(object):
```

Comment:
This works, but I've recently learned there are better ways to mock the django settings. Actually the Django TestCase class contains a context manager for this. So you can do in your test:

```python
class TestIdle(django.tests.TestCase):
    def test_idle(self):
        with self.modify_settings(IDLE_TIMOUT=1000):
            # stuff testing a different idle timout
       # here the original value is back again
```

Good stuff to remember for the future. :)

Reply:
Ooh, good to know, thanks!